ciimage/Dockerfile

```diff
@@ -12,7 +12,9 @@ RUN apt-get -y update && apt-get -y upgrade \
 && apt-get -y install qt4-linguist-tools \
 && apt-get -y install python-dev \
 && apt-get -y install libomp-dev openssh-client \
-&& python3 -m pip install hotdoc codecov
+&& python3 -m pip install hotdoc codecov \
+&& dub fetch urld \
```

Comment:
I wonder whether this should install the `dub` package beforehand, or whether that's not necessary...